scripts/lay-down-os

```diff
@@ -115,39 +109,30 @@ EOF
 fi
 }
 
-format_and_mount()
+mount_fs()
```

Comment:
doesn't format anymore, so i just renamed this.  maybe mount_file_system would be better.  you choose